infrastructure/modules/kms/kms_replica_key_replica.tf

```
@@ -0,0 +1,8 @@
+resource "aws_kms_replica_key" "replica" {
+  count                   = var.is_multi_region ? 1 : 0
+  provider                = aws.us-east-1
+  description             = "Multi-Region replica key"
```

```suggestion
  
  description             = "Multi-Region replica key"
```
empty \n after the meta args (count and provider). Provider should also be above count.